drivers/vmwarevsphere/flags.go

```diff
@@ -0,0 +1,248 @@
+package vmwarevsphere
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/docker/machine/libmachine/drivers"
+	"github.com/docker/machine/libmachine/mcnflag"
+	"github.com/vmware/govmomi/vim25/types"
+)
+
+func (d *Driver) GetCreateFlags() []mcnflag.Flag {
+	return []mcnflag.Flag{
+		mcnflag.IntFlag{
+			EnvVar: "VSPHERE_CPU_COUNT",
+			Name:   "vmwarevsphere-cpu-count",
```

Comment:
I think the other drivers make these name constants, helps avoid typos that would break this

Reply:
ehn